ssb/solr.ecs.boundary.md

```diff
@@ -47,12 +48,15 @@ Rel(aws_cg_alb, cloudgov_router, "proxies requests", "https GET/POST (443)")
 Rel(cloudgov_router, solr_app, "proxies requests", "https GET/POST (443)")
 Rel(solr_app, solr_app_db, "store and read state", "port (3306)")
 Rel(solr_client, solr_ingress, "use SolrCloud instance", "https GET/POST (8193)")
-Rel(solr_ingress, solr_instance, "proxies requests", "http GET/POST (8983)")
-Rel(solr_admin_init_service, solr_instance, "config initialization", "http POST (8983)")
-Rel(solr_app, solr_instance, "provisions", "https GET/POST (443)")
+Rel(solr_ingress, solr_leader, "proxies requests", "http POST (8983)")
+Rel(solr_ingress, solr_followers, "proxies requests", "http GET (8983)")
+Rel(solr_admin_init_service, solr_leader, "config initialization", "http POST (8983)")
+Rel(solr_app, solr_leader, "provisions", "https GET/POST (443)")
 Rel(solr_app, solr_ingress, "provisions", "https GET/POST (443)")
 Rel(solr_app, solr_admin_init_service, "provisions", "https GET/POST (443)")
 Rel(solr_app, solr_efs, "provisions", "https GET/POST (443)")
-Rel(solr_instance, solr_efs, "mounts", "nfs")
+Rel(solr_leader, solr_followers, "distributes updates", "http POST (8983)")
```

Comment:
```suggestion
Rel(solr_leader, solr_followers, "distributes updates", "https POST (443)")
```

Reply:
HTTPS/443 is used for replication. 
https://github.com/GSA/datagov-brokerpak-solr/blob/main/terraform/ecs/provision/follower.tf#L29